### ISSUE TITLE
cdk Failure in step 02_get-kubectl after-deploy (macos, py3.7 can't find yaml)

### ISSUE THREAD

**Issue by kwmonroe** [conjure-up/spells#202]
Deployed cdk to aws.  Step 2 failed with this in the `after-deploy.err`:
```
$ conjure-up --version
conjure-up 2.5.9
$ uname
Darwin

$ cat ~/.cache/conjure-up/canonical-kubernetes/steps/02_get-kubectl/after-deploy.err
Traceback (most recent call last):
  File "/Users/kwmonroe/.cache/conjure-up/canonical-kubernetes/steps/02_get-kubectl/update_kube_config.py", line 4, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

**Comment by battlemidget** [comment]
Very odd, we have the beta referenced in the formula.

I'll need to see if we're missing something the tests aren't catching

**Comment by battlemidget** [comment]
Looks like pyyaml 3.13 will be released with 3.7 support, so we'll want to update the brew formula when that is released, hopefully today or tomorrow

**Comment by spjz** [comment]
Still having this issue as of today with a fresh install of Ubuntu 16.04 Server LTS, with current snaps for lxd and conjure-up.

**Comment by kwmonroe** [comment]
Oof, sorry I left this open so long.  Confirmed fixed in `conjure-up 2.6.0` on macos.  This fixed me a few months ago:

https://github.com/Homebrew/homebrew-core/commit/6bc6dd64a1f810db99d4acfcb3cd7accd14ec2ea

@spjz, this was an issue with homebrew/macos deps, so it won't be the same root cause for ubuntu.
